packages/stream_video_flutter/lib/stream_video_flutter_background.dart

```diff
@@ -41,18 +41,20 @@ mixin StreamVideoFlutterBackground {
     );
   }
 
-  static Future<bool> stopService(ServiceType type) async {
```

Comment:
Is there an option to make this `callId` optional and check if there is only 1 call running? Would prevent a breaking change and makes code slightly easier for when you don't allow for multiple calls.

Or maybe to make it more clear we could add a different method like this:

```dart
static Future<bool> stopService(ServiceType type) {
  final callCid = getCurrentCall();
  return stopServiceForCall(type, callCid: callCid);
}

static Future<bool> stopServiceForCall(ServiceType type, {required String callCid}) async {
...
}